src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -64,23 +68,48 @@ void DoorLockServer::InitServer(chip::EndpointId endpointId)
 {
     emberAfDoorLockClusterPrintln("Door Lock cluster initialized at endpoint #%" PRIu16, endpointId);
 
-    SetLockState(endpointId, DlLockState::kLocked);
+    SetLockState(endpointId, DlLockState::kLocked, DlOperationSource::kUnspecified);
```

Comment:
I couldn't find where we set the lock to `kLocked` or `kUnlocked` from the app, at least explicitly. Is that expected?